### ISSUE TITLE
Auth fails if login contains query params

### ISSUE THREAD

**Issue by u12206050** [directus/directus#9749]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

When trying to login using SSO this happens when the url looks like:

`/admin/login` WORKS
`/admin/login?anything` FAILS

### To Reproduce

Login using SSO make sure the is a query param eg. `?test` at the end of the url

### Errors Shown

{"errors":[{"message":"Invalid user credentials.","extensions":{"code":"INVALID_CREDENTIALS"}}]}

### What version of Directus are you using?

9.0.1

### What version of Node.js are you using?

16.4.0

### What database are you using?

mysql 8

### What browser are you using?

chrome

### What operating system are you using?

osx

### How are you deploying Directus?

GCP

**Comment by u12206050** [comment]
Seems to also be a problem on version 9.0.0

**Comment by joselcvarela** [comment]
Hello @u12206050 
Are you experiencing this issue with the API request? I am not sure I understand what are you experiencing.

I made this request:
```http
POST http://localhost:8055/auth/login?test
Content-Type: application/json
{"email":"xxxx", "password":"xxxx"}
```

And seems to work as expected.

**Comment by u12206050** [comment]
Hi, after updating to 9.1.2 it works, and checking back it seems to have been fixed in v9.0.1. Thanks
